### PR TITLE
SCC post registration

### PR DESCRIPTION
**Pull request by dzedro**
Changed tab to alt-c, tab doesn't work for post registration,
because email field has shortcut alt-m, not alt-e for installation registration.

### CODE REVIEW

**Comment by dzedro** [comment]
http://10.100.98.90/tests/2074/modules/scc_postregistration/steps/7

**Comment by lnussel** [comment]
doesn't this duplicate https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/zypper_ar_scc.pm ?

File: tests/x11/scc_postregistration.pm [hunk 1]
```diff
@@ -0,0 +1,22 @@
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    x11_start_program('xterm');
+    become_root;
+    assert_script_run 'rm -rf /etc/zypp/credentials.d /etc/zypp/repos.d /etc/zypp/services.d';  # delete existing registration data
```
Comment (lnussel) [inline tests/x11/scc_postregistration.pm]: also removes repo that points to iso

**Comment by dzedro** [comment]
updated http://10.100.98.90/tests/2155/modules/scc_postregistration/steps/1

**Pull request merged**